Improve the backup description based on a comment from sales (#603)
owncloud/docs-ocis: modules/ROOT/pages/maintenance/b-r/backup.adoc

For details on supported filesystems, see: xref:prerequisites/prerequisites.adoc#filesystems-and-shared-storage[Filesystems and Shared Storage].

Depending on how this data is stored, backup strategies and tasks will differ, see the sections below.

You have to fully stop the Infinite Scale instance to create a consistent state necessary for performing the backup. In the following scenarios, this state of consistency is a prerequisite for all upcoming tasks. To avoid redundancy, this step will not be repeatedly written down every time. When backup has finished, Infinite Scale can be started again.

> There are two general approaches how to achieve a backup:

Reviewer: ```suggestion
There are two general approaches to achieve a backup:
```

> * When the underlying filesystems of the storages used provide snapshot capabilities, creating the base for the backup will only take a few seconds. This is the preferred method as the Infinite Scale instance can be started post creating the necessary snapshots again. To prevent data loss in case of hardware failures, it is recommend to copy the snapshots to a secondary storage or use a backup software of your choice that accesses the snapshots for further processing.

Reviewer: ```suggestion
* When the underlying filesystems of the storages used provide snapshot capabilities, creating the base for the backup will only take a few seconds. This is the preferred method as the Infinite Scale instance can be started again directly after creating the necessary snapshots. To prevent data loss in case of hardware failures, it is recommend to copy the snapshots to a secondary storage or use backup software of your choice that accesses the snapshots for further processing.
```

> * In case snapshot capabilities are not avialable or cant be used, a backup software of your choice can be taken to backup Infinite Scale.

Reviewer: ```suggestion
* If snapshot capabilities are not available or can't be used, backup software of your choice can be used to backup Infinite Scale.
```
